homeassistant/components/notify/pushbullet.py

```diff
@@ -89,7 +89,7 @@ def send_message(self, message=None, **kwargs):
 
         if not targets:
             # Backward compatibility, notify all devices in own account
-            self._push_data(filepath, message, title, url)
+            self._push_data(filepath, message, title, self.pushbullet, url)
```

Comment:
In this backward compatible case, the url and pusher are in the wrong order (should be swapped). I can submit a pull request in a couple days if needed.

Reply:
It is already fixed: https://github.com/home-assistant/home-assistant/pull/9200